src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -1069,6 +1099,60 @@ public static void loadGameCharacterVariablesFromXML(GameCharacter character, St
 					nameElement.getAttribute("nameFeminine")));
 		}
 		
+		// Colours:
+		if(element.getElementsByTagName("colours").getLength() > 0)
+		{
```

Comment:
maybe
```
private String[] colorTypes = {"primaryColour", "secondaryColour", "underwearColour", "leatherColour", "denimColour", "metalColour"};
for (String ctp: colorTypes)
{
	try
	{
		character.secondaryColour = Colour.valueOf(colourElement.getAttribute(ctp));
	}
	catch(IllegalArgumentException e)
	{
		System.err.println("Warning: Invalid " + ctp);
	}
}
```

Reply:
I'm not sure how to make that work, since we are setting six different variables here.